clj-refactor.el

```diff
@@ -2199,6 +2200,12 @@ the alias in the project."
           (-when-let (long (cljr--aget cljr-magic-require-namespaces short))
             (list short (list long))))))))
 
+(defun cljr--in-sans-ns-keyword-p ()
```

Comment:
ok I rather prefer the `-p` style preds. But really don't want to go down the rabbit hole of renaming all our `foo?` style preds in this PR.
- will create a separate PR replacing all our own `?` predicates with `-p` ones
- keep this fn `-p` style